sources/packages/web/src/components/common/EnvironmentBanner.vue

```diff
@@ -0,0 +1,65 @@
+<template>
+  <div v-if="!isProduction" class="environment-banner">
+    <span class="environment-banner-text">{{ bannerText }}</span>
+  </div>
+</template>
+
+<script lang="ts">
+import { AppConfigService } from "@/services/AppConfigService";
+import { defineComponent, computed, ref, onMounted } from "vue";
+
+export default defineComponent({
+  setup() {
+    const appEnv = ref("");
+    const isProduction = ref(false);
+
+    onMounted(async () => {
+      // Get environment from env variable APP_ENV | dev, test, staging, production
+      const config = await AppConfigService.shared.config();
```

Comment:
Appenv can be directly imported. 

```ts
const { appEnv } = await AppConfigService.shared.config();
      isProduction.value = appEnv === "production";
```

Reply:
It is because you are using the same variable name as config value in ref.

```ts
const environment = ref("");
const { appEnv } = await AppConfigService.shared.config();
environment.value = appEnv;
```